### PR TITLE
WIP: Allow TranslatableMarkup and add italic style in wrapTextDecorations.

### PR DESCRIPTION
**Pull request by dipakmdhrm**
#376 

### Update:
- Updated `wrapTextDecorations()` to allow `TranslatableMarkup`.
- Updated `wrapTextDecorations()` to allow italic text.

italic

### CODE REVIEW

File: web/modules/custom/server_general/server_general.module [hunk 1]
```diff
@@ -20,7 +20,7 @@ use Drupal\server_general\ArabicNumberConverter;
  * That can be based on form.html.twig.
  */
 function server_general_theme_suggestions_alter(array &$suggestions, array $variables) {
-  if (isset($variables['element']) && isset($variables['element']['#type']) && $variables['element']['#type'] == 'form') {
+  if (isset($variables['element']) && is_array($variables['element']) && isset($variables['element']['#type']) && $variables['element']['#type'] == 'form') {
```
Comment (dipakmdhrm) [inline web/modules/custom/server_general/server_general.module:23]: Without this we'd get the error: 

`Cannot use object of type Drupal\Core\StringTranslation\TranslatableMarkup as array in server_general_theme_suggestions_alter() (line 32 of modules/custom/server_general/server_general.module).`

**Comment by amitaibu** [comment]
@dipakmdhrm needs a re-roll

**Comment by amitaibu** [comment]
Let's put on hold, as we're doing a bigger change in https://github.com/Gizra/drupal-starter/pull/388

**Comment by amitaibu** [comment]
Closed in favor of https://github.com/Gizra/drupal-starter/pull/391

**Pull request closed without merging**